composer.json

```diff
@@ -62,7 +62,7 @@
         "zendframework/zend-eventmanager": "^3.0.0",
         "zendframework/zend-feed": "^2.9.0",
         "zendframework/zend-form": "^2.10.0",
-        "zendframework/zend-http": "^2.6.0",
+        "zendframework/zend-http": "2.10.0",
```

Comment:
@engcom-Foxtrot why did you replaced dependency for zend-http? I think it's not correct

Reply:
Hello, @ihor-sviziev. Seems like 2.10.1 makes some backward incompatible changes.